### PR TITLE
Yandex.Navigator support

### PR DESCRIPTION
**Pull request by frkncngz**
This adds Yandex.Navigator support.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,9 +23,9 @@ Firstly insert all Third Party Apps' URL schemes to your **Info.plist**:
  - comgooglemaps ([Google Maps App](https://itunes.apple.com/us/app/google-maps-real-time-navigation/id585027354?mt=8))
  - navigon ([Navigon App](https://itunes.apple.com/us/app/navigon-usa/id384680007?mt=8))
  - transit ([Transit App](https://itunes.apple.com/us/app/transit-app-real-time-tracker/id498151501?mt=8))
- - waze ([Waze App](https://itunes.apple.com/us/app/waze-gps-navigation-maps-social/id323229106?mt=8))
```
Comment (NSDavidObject) [inline README.md:26]: This should be reverted

**Comment by NSDavidObject** [comment]
Looks good, just a minor comment. Please also include Yandex in the tests.

**Comment by frkncngz** [comment]
Yandex is added to the tests and Waze is back in the readme ✌️

**Comment by NSDavidObject** [comment]
@frkncngz thanks for the updates, and for getting this in! :)

**Pull request merged**